dumpling/export/dump.go

```diff
@@ -1637,3 +1648,8 @@ func (d *Dumper) renewSelectTableRegionFuncForLowerTiDB(tctx *tcontext.Context)
 
 	return nil
 }
+
+func (d *Dumper) newTaskTableData(meta TableMeta, data TableDataIR, currentChunk, totalChunks int) *TaskTableData {
+	d.metrics.totalChunks.Add(1)
+	return NewTaskTableData(meta, data, currentChunk, totalChunks)
```

Comment:
I found a direct call to `NewTaskTableData` in `buildConcatTask` without increase the `totalChunks`, but I'm not familiar with the logic. Should we also wrap for it?

Reply:
good catch, we should wrap for it. And we should also decrease the `totalChunks` after `concatTask` because the concated task is not sent to the real `taskChan`, what do you think?

Reply:
in fact I'm not sure what's a "concated task" 😂 
/cc @lichunzhu

Reply:
A concacted task is used to dump several ranges who belongs to one table with one connection one concurrency. It's used when users don't specify `rows` when dumping TiDB. It's used to save TiDB memory usage.